src/components/AnonymousChat/child/ChannelFollowButton.tsx

```diff
@@ -57,10 +57,10 @@ const styles = StyleSheet.create({
     fontSize: 12
   },
   textFollow: {
-    color: colors.white
+    color: COLORS.white
   },
   textFollowing: {
-    color: colors.darkBlue
+    color: COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color for the follow and following states have been updated to use the new `COLORS.white` and `COLORS.signed_primary` values respectively. This change is consistent with the PR description and seems to be part of a larger refactoring effort to centralize color management in the application.
```diff
-    color: colors.white
+    color: COLORS.white

-    color: colors.darkBlue
+    color: COLORS.signed_primary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->